benchmarks/benchmark_serving.py

```diff
@@ -576,7 +601,10 @@ def main(args: argparse.Namespace):
 
   if args.warmup_first:
     print("Warm up start:")
-    warmup_requests = list(sample_warmup_requests(input_requests)) * 2
+    if args.full_warmup:
```

Comment:
What is the full_warmup here? Why does the `warmup_requests = list(sample_warmup_requests(input_requests)) * 2` not work?

Reply:
Full warmup gets better perf result than the existing warmup, shared with you the detail results offline